### ISSUE TITLE
Solargraph started to give "[FrozenError] can't modify frozen NilClass: nil"

### ISSUE THREAD

**Issue by gencer** [castwide/solargraph#389]
## Note before read below

I was using solargraph's `master` branch and I forgot to revert back to stable version. I thought this error was on stable version (0.39.x). It seems it throws this exception on master branch. I just want to report anyway. If you already know this issue you may close this issue.

### Version:
0.40.0 (master)

### Output:

```
[Error - 10:31:26 AM] Server initialization failed.
  Message: [FrozenError] can't modify frozen NilClass: nil
  Code: -32603 
```

### Developer Console:

```
[Extension Host] ruby symbols active v 0.1.4
mainThreadExtensionService.ts:65 [[object Object]]u[t] is not a constructor
$onExtensionRuntimeError @ terminalInstanceService.ts:31
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
mainThreadExtensionService.ts:66 TypeError: u[t] is not a constructor
	at c._runLinters (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at c.run (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at t.LintCollection.run (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at r (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:856
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at b.acceptDocumentsAndEditorsDelta (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:743)
	at b.$acceptDocumentsAndEditorsDelta (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:741)
	at m._doInvokeHandler (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:895)
	at m._invokeHandler (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:895)
	at m._receiveRequest (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:893)
	at m._receiveOneMessage (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:892)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:890
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at v.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:256)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:1084
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at v.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:256)
	at t.PersistentProtocol._receiveMessage (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:261)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:258
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at p.acceptChunk (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:253)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:253
	at Socket.t (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:263)
	at Socket.emit (events.js:223)
	at addChunk (_stream_readable.js:309)
	at readableAddChunk (_stream_readable.js:290)
	at Socket.Readable.push (_stream_readable.js:224)
	at TCP.onStreamRead (internal/stream_base_commons.js:181)
$onExtensionRuntimeError @ mainThreadExtensionService.ts:66
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
console.ts:137 [Extension Host] Solargraph is listening PORT=60399 PID=2346036
console.ts:137 [Extension Host] [WARN] Failed to load plugin 'runtime'
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/colorize-0.8.1/.yardoc is too large to process (30406610 bytes)
mainThreadExtensionService.ts:65 [[object Object]]u[t] is not a constructor
$onExtensionRuntimeError @ terminalInstanceService.ts:31
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
mainThreadExtensionService.ts:66 TypeError: u[t] is not a constructor
	at c._runLinters (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at c.run (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at t.LintCollection.run (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at r (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:856
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at b.acceptDocumentsAndEditorsDelta (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:743)
	at b.$acceptDocumentsAndEditorsDelta (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:741)
	at m._doInvokeHandler (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:895)
	at m._invokeHandler (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:895)
	at m._receiveRequest (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:893)
	at m._receiveOneMessage (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:892)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:890
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at v.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:256)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:1084
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at v.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:256)
	at t.PersistentProtocol._receiveMessage (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:261)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:258
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at p.acceptChunk (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:253)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:253
	at Socket.t (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:263)
	at Socket.emit (events.js:223)
	at addChunk (_stream_readable.js:309)
	at readableAddChunk (_stream_readable.js:290)
	at Socket.Readable.push (_stream_readable.js:224)
	at TCP.onStreamRead (internal/stream_base_commons.js:181)
$onExtensionRuntimeError @ mainThreadExtensionService.ts:66
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/capistrano-3.14.1/.yardoc is too large to process (20574058 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/airbrussh-1.4.0/.yardoc is too large to process (20392649 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/sshkit-1.21.0/.yardoc is too large to process (20328957 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/net-ssh-6.1.0/.yardoc is too large to process (20095004 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/net-scp-3.0.0/.yardoc is too large to process (20163064 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/unf-0.1.4/.yardoc is too large to process (30470821 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/unf_ext-0.0.7.7/.yardoc is too large to process (30468136 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/domain_name-0.5.20190701/.yardoc is too large to process (30889427 bytes)
console.ts:137 [Extension Host] [WARN] Yardoc at /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/doc/http-cookie-1.0.3/.yardoc is too large to process (31012050 bytes)
console.ts:137 [Extension Host] [WARN] Error processing request: [FrozenError] can't modify frozen NilClass: nil
console.ts:137 [Extension Host] [WARN] /home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/yard_map.rb:243:in `instance_variable_set'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/yard_map.rb:243:in `process_requires'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/yard_map.rb:92:in `change'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/api_map.rb:122:in `catalog'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/library.rb:23:in `initialize'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/library.rb:377:in `new'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/library.rb:377:in `load'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/host.rb:278:in `prepare'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/host.rb:295:in `block in prepare_folders'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/host.rb:294:in `each'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/host.rb:294:in `prepare_folders'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/message/initialize.rb:14:in `block in process'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/2.7.0/benchmark.rb:293:in `measure'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/message/initialize.rb:10:in `process'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/host.rb:101:in `receive'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/adapter.rb:44:in `process'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/adapter.rb:17:in `block in opening'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/data_reader.rb:59:in `parse_message_from_buffer'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/data_reader.rb:35:in `block in receive'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/data_reader.rb:30:in `each_char'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/data_reader.rb:30:in `receive'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solargraph-0.40.0/lib/solargraph/language_server/transport/adapter.rb:27:in `receiving'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/client.rb:63:in `tick'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/server/tcpip.rb:76:in `update'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/2.7.0/observer.rb:197:in `block in notify_observers'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/2.7.0/observer.rb:196:in `each'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/2.7.0/observer.rb:196:in `notify_observers'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/client.rb:121:in `read_input'/home/gencer/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/backport-1.1.2/lib/backport/client.rb:102:in `block in run_input_thread'
notificationsAlerts.ts:40 [FrozenError] can't modify frozen NilClass: nil
onDidChangeNotification @ notificationsAlerts.ts:25
(anonymous) @ notificationsAlerts.ts:25
fire @ event.ts:535
addNotification @ notifications.ts:204
notify @ notificationService.ts:53
(anonymous) @ mainThreadMessageService.ts:21
_showMessage @ mainThreadMessageService.ts:21
$showMessage @ mainThreadMessageService.ts:21
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
console.ts:137 [Extension Host] Error starting Solargraph socket provider Error: [FrozenError] can't modify frozen NilClass: nil
	at new ResponseError (/home/gencer/.vscode-server/extensions/castwide.solargraph-0.21.1/node_modules/vscode-languageclient/node_modules/vscode-jsonrpc/lib/messages.js:46:28)
	at handleResponse (/home/gencer/.vscode-server/extensions/castwide.solargraph-0.21.1/node_modules/vscode-languageclient/node_modules/vscode-jsonrpc/lib/main.js:430:48)
	at processMessageQueue (/home/gencer/.vscode-server/extensions/castwide.solargraph-0.21.1/node_modules/vscode-languageclient/node_modules/vscode-jsonrpc/lib/main.js:258:17)
	at Immediate.<anonymous> (/home/gencer/.vscode-server/extensions/castwide.solargraph-0.21.1/node_modules/vscode-languageclient/node_modules/vscode-jsonrpc/lib/main.js:242:13)
	at processImmediate (internal/timers.js:439:21)
notificationsAlerts.ts:40 Failed to start Solargraph: Error: [FrozenError] can't modify frozen NilClass: nil
onDidChangeNotification @ notificationsAlerts.ts:25
(anonymous) @ notificationsAlerts.ts:25
fire @ event.ts:535
addNotification @ notifications.ts:204
notify @ notificationService.ts:53
(anonymous) @ mainThreadMessageService.ts:21
_showMessage @ mainThreadMessageService.ts:21
$showMessage @ mainThreadMessageService.ts:21
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
mainThreadExtensionService.ts:65 [[object Object]]u[t] is not a constructor
$onExtensionRuntimeError @ terminalInstanceService.ts:31
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
mainThreadExtensionService.ts:66 TypeError: u[t] is not a constructor
	at c._runLinters (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at c.run (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at t.LintCollection.run (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at r (/home/gencer/.vscode-server/extensions/rebornix.ruby-0.27.0/dist/client/ruby.js:9)
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:856
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at b.acceptDocumentsAndEditorsDelta (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:743)
	at b.$acceptDocumentsAndEditorsDelta (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:741)
	at m._doInvokeHandler (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:895)
	at m._invokeHandler (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:895)
	at m._receiveRequest (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:893)
	at m._receiveOneMessage (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:892)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:890
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at v.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:256)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:1084
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at v.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:256)
	at t.PersistentProtocol._receiveMessage (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:261)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:258
	at l.fire (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:45)
	at p.acceptChunk (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:253)
	at /home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:253
	at Socket.t (/home/gencer/.vscode-server/bin/d2e414d9e4239a252d1ab117bd7067f125afd80a/out/vs/server/remoteExtensionHostProcess.js:263)
	at Socket.emit (events.js:223)
	at addChunk (_stream_readable.js:309)
	at readableAddChunk (_stream_readable.js:290)
	at Socket.Readable.push (_stream_readable.js:224)
	at TCP.onStreamRead (internal/stream_base_commons.js:181)
$onExtensionRuntimeError @ mainThreadExtensionService.ts:66
_doInvokeHandler @ rpcProtocol.ts:401
_invokeHandler @ rpcProtocol.ts:332
_receiveRequest @ rpcProtocol.ts:278
_receiveOneMessage @ rpcProtocol.ts:220
(anonymous) @ rpcProtocol.ts:93
fire @ event.ts:535
fire @ ipc.net.ts:407
_receiveMessage @ ipc.net.ts:755
(anonymous) @ ipc.net.ts:630
fire @ event.ts:535
acceptChunk @ ipc.net.ts:224
(anonymous) @ ipc.net.ts:149
t @ ipc.net.ts:892
emit @ events.js:223
addChunk @ _stream_readable.js:309
readableAddChunk @ _stream_readable.js:290
Readable.push @ _stream_readable.js:224
onStreamRead @ internal/stream_base_commons.js:181
```

**Comment by pelletencate** [comment]
I have the very same issue in VSCodium, MRI 2.7.2 on macOS 11.

If I run `solargraph stdio` on my terminal, it works fine.

**Comment by castwide** [comment]
This should be fixed in the master branch. I'll release the fix in v0.40.0.

**Comment by castwide** [comment]
Released in version 0.40.0.

One thing to note: this bug was related to mapping the YAML module, which currently does not work in Ruby 2.7 for reasons I haven't figured out yet, but it no longer causes an error.
